classes/AuthManager.php

```diff
@@ -126,4 +127,22 @@ public function convertGuestToUser($user, $credentials, $activate = false)
 
         return $this->user = $user;
     }
+
+    // replaces Redirector::guest to use a seperate session variable in place of url.intended (used on the backend)
```

Comment:
Proper doc blocks please, you can copy over the ones from Redirector and modify accordingly